test/test.bats

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env bats
```

Comment:
What is the purpose of this file? It seems to contain lot's of test code of external libs which seems out of scope of this lib

Reply:
That's just a test file I've used to check the highlights of each functions.
The code came from the libs repositories (copied from test files for each function).
I didn't find an easy way to automate grammar coloration testing.
Not mandatory to keep in sources.

Reply:
Then trim it down:
- Use every/most functions that should be highlighted. Only syntactically correct (no need for valid tests, keep it short)
- Use (some) functions in strings (in code and test name) and comments
- Use variables
- Opt: Add comments what to look for

Reply:
You'r right, I was a bit quick and dirty.
I've jsut push an updated file.